src/registrar/templates/domain_dnssec.html

```diff
@@ -0,0 +1,68 @@
+{% extends "domain_base.html" %}
+{% load static field_helpers url_helpers %}
+
+{% block title %}DNSSEC | {{ domain.name }} | {% endblock %}
+
+{% block domain_content %}
+
+  <h1>{% if dnssec_enabled %}Set up {% endif %}DNSSEC</h1>
+
+  <p>DNSSEC, or DNS Security Extensions, is additional security layer to protect your website. Enabling DNSSEC ensures that when someone visits your website, they can be certain that it's connecting to the correct server, preventing potential hijacking or tampering with your domain's records.</p>
+
+  <form class="usa-form usa-form--large" method="post">
+    {% csrf_token %}
+    {% if has_dnssec_records %}
+    <div class="usa-alert usa-alert--info usa-alert--slim margin-bottom-3">
+      <div class="usa-alert__body">
+      In order to fully disable DNSSEC on your domain, you will need to work with your DNS provider to remove your DNSSEC-related records from your zone.
+      </div>
+    </div>
+    <a
+      href="#toggle-dnssec-alert"
+      class="usa-button"
+      aria-controls="toggle-dnssec-alert"
+      data-open-modal
+      >Disable DNSSEC</a
+    >
+    {% elif dnssec_enabled %}
+    <div id="add-records">
+      <h2> Add DS Records </h2>
+      <p>In order to enable DNSSEC and add Delegation Signer (DS) records, you must first configure it with your DNS hosting service. Your configuration will determine whether you need to add DS Data or Key Data. Contact your DNS hosting provider if you are unsure which record type to add.</p>
+      <p>
+        <a href="{% url 'domain-dns-dnssec-dsdata' pk=domain.id %}" class="usa-button usa-button--outline">Add DS Data</a>
+        <a href="{% url 'domain-dns-dnssec-keydata' pk=domain.id %}" class="usa-button usa-button--outline">Add Key Data</a>
+        <button
+          type="submit"
+          class="usa-button usa-button--unstyled"
+          name="cancel_dnssec"
+          id="cancel_dnssec"
+      >Cancel</button>
+      </p>
+    </div>
+    {% else %}
+    <div id="enable-dnssec">
+      <div class="usa-alert usa-alert--info usa-alert--slim margin-bottom-3">
+        <div class="usa-alert__body">
+        It is strongly recommended that you do not enable this unless you fully understand DNSSEC and know how to set it up properly. If you make a mistake, it could cause your domain name to stop working.
```

Comment:
Change to:

> It is strongly recommended that you only enable DNSSEC if you know how to set it up properly at your hosting service. If you make a mistake, it could cause your domain name to stop working.